clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/cache/Keys.groovy

```diff
@@ -69,8 +69,9 @@ class Keys {
         result << [
             application: parts[3],
             account: parts[2],
-            name: parts[4],
-            cluster: parts[4],
+            name: parts[5],
+            cluster: parts[5],
+            category: parts[4],
```

Comment:
It mirrors "category" in deck, where it's becoming a more pervasive way to distinguish between server groups and jobs. I'll leave a comment here.

.. `clusterF$%`, really?